pnpm-workspace.yaml

```diff
@@ -0,0 +1,20 @@
+packages:
+  - /*
+  - packages/*
+
+onlyBuiltDependencies:
+  - '@ckeditor/ckeditor5-dev-tests'
+  - '@ckeditor/ckeditor5-mermaid'
+  - '@ocular-d/vale-bin'
+  - '@parcel/watcher'
+  - '@swc/core'
+  - core-js
+  - esbuild
+  - hexo-util
+  - puppeteer
+  - sharp
+
+shellEmulator: true
```

Comment:
This option enables bash-like shell. It simplifies cross-platform scripting, allowing for scripts like this (which will also work on Windows):

```json
"scripts": {
  "test": "NODE_ENV=test node test.js"
}
```

More information: https://pnpm.io/cli/run#shellemulator

Reply:
The downside is that it tries to be smart about glob patterns like `**/node_modules` which in the following case crashed v8:

```
pnpx rimraf --glob **/node_modules
```

The workaround was to wrap the glob pattern in quotes:

```
pnpx rimraf --glob \"**/node_modules\"
```